ScenePlayer/res/layout/recording_player.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-	xmlns:rjdj="http://schemas.android.com/apk/res/me.rjdj.rjdjroid"
```

Comment:
If xmlns is needed anywhere for something similar, it should point to the generic "res-auto" schema for gradle-build compatibility.